docs/user/editor/calendars.md

```diff
@@ -19,8 +19,8 @@ Choose a calendar from the list to begin editing. To create a new calendar, clic
 
 Schedule exceptions allow users to define days where special services should replace the regularly operating calendars. To create a schedule exception, click the `Exceptions` tab and then click `+ New exception` (replaces the "New calendar" button).
 
-<div class="img-center"> 
-![create exception](../../gif/schedule-exception.gif)
+<div style = "text-align:center"> 
```

Comment:
Please remove the spaces around `=`. Also, the image tag should be `<img src="path/to/image" />` (not ending with `</img>`).